packages/paste-core/components/progress-steps/__tests__/index.spec.tsx

```diff
@@ -69,6 +77,9 @@ describe("ProgressSteps", () => {
       expect(completeProfile?.dataset.pasteElement).toEqual("PROGRESS_STEP_CURRENT");
       expect(addFriends?.dataset.pasteElement).toEqual("PROGRESS_STEP_INCOMPLETE");
       expect(wrapper?.dataset.pasteElement).toEqual("PROGRESS_STEPS");
+      expect(
+        render(<WithContent />).baseElement.querySelector("[data-paste-element='PROGRESS_STEP_CONTENT']"),
+      ).toBeInTheDocument();
```

Comment:
suggestion(non-blocking): ideally if we are doing a render of a different instance this should be in a new test